xbmc/utils/FileUtils.h

```diff
@@ -21,6 +21,10 @@ class CFileUtils
   static bool DeleteItem(const std::shared_ptr<CFileItem>& item);
   static bool DeleteItem(const std::string &strPath);
   static bool Exists(const std::string& strFileName, bool bUseCache = true);
+  
+  /** \brief Delete a file with confirmation unless setting is off.
+  */
```

Comment:
bad formatting style,
and since this must used only for audio/video/music you should specify it,
because this class is generic
```c++
/*
 * \brief Deleting a file with confirmation according to settings, this applies only to audio/video/music files.
 */
```

Reply:
Agree to formatting, not to charging the description. The implementation deletes whatever it gets. The fact for what is currently used is nothing for the description of the function. Keep as is, please.